### DISCUSSION TITLE
Using an explicit version with "Alpine" reports an error with versions for AIX?

### DISCUSSION THREAD

**Discussion by johanneskastl** [ansible/ansible-lint#3868] [Q&A]
https://github.com/johanneskastl/ansible-role-doas_configuration/actions/runs/6637641821

The role contains the following:

```
  platforms:
    - name: Alpine
      versions:
        - '3.18'
```

The error given by ansible-lint contains the versions that are defined in the AIX block:

```
$.galaxy_info.platforms[0].versions[0] '3.18' is not one of ['6.1', '7.1', '7.2', 'all']. 
```

https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/schemas/meta.json#L13

```
    "AIXPlatformModel": {
      "properties": {
        "name": {
          "const": "AIX",
          "title": "Name",
          "type": "string"
        },
        "versions": {
          "default": "all",
          "items": {
            "enum": ["6.1", "7.1", "7.2", "all"],
            "type": "string"
          },
          "type": "array"
        }
      },
      "title": "AIXPlatformModel",
      "type": "object"
    },
```

If I understand the schema properly, `all` is the only value allowed/defined for Alpine. Shouldn't ansible-lint then report this, instead of the AIX versions?

**Comment by alessfg** [comment]
I am running into this same issue, although tbqh platform validation should probably be removed altogether given it's no longer supported by Galaxy. What do you think @ssbarnea?
